lib/rex/post/meterpreter/extensions/stdapi/fs/dir.rb

```diff
@@ -209,8 +210,8 @@ def Dir.download(dst, src, recursive = false, force = true, &stat)
       if (src_stat.file?)
         stat.call('downloading', src_item, dst_item) if (stat)
         begin
-          client.fs.file.download(dst_item, src_item)
-          stat.call('downloaded', src_item, dst_item) if (stat)
+          result = client.fs.file.download_file(dst_item, src_item)
+          stat.call(result, src_item, dst_item) if (stat)
```

Comment:
I see which ```client.fs.file.download_file``` will return 'download' instead of 'downloaded' when returning correctly. Is it okey? I mean, before this change 'downloaded' was being hardcoded. Was it incorrect before this change?